Dockerfile

```
@@ -38,17 +37,15 @@ WORKDIR /srv
 RUN chown -R node:node /srv/ && \
     chmod 0750 /srv/
 
-COPY --chown=node:node package.json yarn.lock /srv/
-RUN chmod 0640 /srv/package.json /srv/yarn.lock
+COPY --chown=node:node . /srv
```

```suggestion
COPY --chown=node:node package.json yarn.lock .yarnrc.yml /srv/
RUN chmod 0640 /srv/package.json /srv/yarn.lock /srv/.yarnrc.yml
```
This approach is a bit better optimized when it comes to Docker Layer Caching.

  - Layer 1: Copy only package.json, yarn.lock, .yarnrc.yml (files that rarely change)
  - Layer 2: Run yarn install --immutable (expensive operation, now cached)
  - Layer 3: Copy all source code (including .env.development)
  - Layer 4: Build the application
  
  Without this, any modification to source code, even if the packages were not changed would mean the whole layer must be rebuild. 
  With this change, the dependency installation is cached and don't need to be rebuild.